.github/configuration/upgrade-test-configuration.yml

```diff
@@ -106,6 +106,18 @@ upgrade-version: [{
     }
   ],
 },
+{
+  upgrade-path: [
+    {
+      version: '14.19',
```

Comment:
Let's update the existing 14.18->latest to 14.19->latest.

Reply:
done, but i am curious why?